app/services/candidate_interface/cancel_unsubmitted_application_at_end_of_cycle.rb

```diff
@@ -7,6 +7,8 @@ def initialize(application_form)
     def call
       @application_form.application_choices.unsubmitted.each do |application_choice|
         ApplicationStateChange.new(application_choice).reject_at_end_of_cycle!
+      rescue ActiveRecord::RecordInvalid => e
+        Rails.logger.warn "Unable to reject application #{application_choice.id}. Error: #{e}"
```

Comment:
I assume this will appear in Sentry?

Reply:
It won't because we've rescued it. But that's fine because we're watching it and I'll be able to find the application choice that hasn't been transitioned and work out what's going on. It is just noise if we send it to sentry.